.env.staging

```
@@ -3,3 +3,7 @@ API_URL=https://api-staging.example.com
 VAR_NUMBER=456
 VAR_BOOL=false
 SECRET_KEY=staging-secret-key
+
+# Supabase Configuration
+SUPABASE_URL=https://mgbekkpswaizzthgefbc.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im1nYmVra3Bzd2Fpenp0aGdlZmJjIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTYxMjM0MjIsImV4cCI6MjA3MTY5OTQyMn0.xLyrORmBLCBDYnoJg5krYP3Qav4yDBUC1EGqtZ3tRFs
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)